api/apps/api/test/scenarios/get-scenario-blm-calibration-results.fixtures.ts

```diff
@@ -0,0 +1,81 @@
+import { ScenarioType } from '@marxan-api/modules/scenarios/scenario.api.entity';
+import { HttpStatus } from '@nestjs/common';
+import * as request from 'supertest';
+import { BlmCalibrationRunResultDto } from '../../src/modules/scenarios/dto/scenario-blm-calibration-results.dto';
+import { GivenProjectExists } from '../steps/given-project';
+import { GivenUserIsLoggedIn } from '../steps/given-user-is-logged-in';
+import { bootstrapApplication } from '../utils/api-application';
+import { OrganizationsTestUtils } from '../utils/organizations.test.utils';
+import { ProjectsTestUtils } from '../utils/projects.test.utils';
+import { ScenariosTestUtils } from '../utils/scenarios.test.utils';
+
+export const getFixtures = async () => {
+  const app = await bootstrapApplication();
+  const token = await GivenUserIsLoggedIn(app);
+
+  const { projectId, organizationId } = await GivenProjectExists(
+    app,
+    token,
+    {
+      countryCode: 'AGO',
+      name: `Project name ${Date.now()}`,
+    },
+    {
+      name: `Org name ${Date.now()}`,
+    },
+  );
+
+  await ProjectsTestUtils.generateBlmValues(app, projectId);
+  let scenarioId: string;
+  const blmRange = [0, 100];
+
+  return {
+    cleanup: async () => {
+      await ProjectsTestUtils.deleteProject(app, token, projectId);
+      await ScenariosTestUtils.deleteScenario(app, token, scenarioId);
+      await OrganizationsTestUtils.deleteOrganization(
+        app,
+        token,
+        organizationId,
+      );
+      await app.close();
+    },
+    GivenScenarioWasCreated: async () => {
+      const result = await ScenariosTestUtils.createScenario(app, token, {
+        name: `Test scenario`,
+        type: ScenarioType.marxan,
+        projectId,
+      });
+      scenarioId = result.data.id;
+    },
+    WhenBlmCalibrationIsLaunched: async () =>
+      await request(app.getHttpServer())
+        .post(`/api/v1/scenarios/${scenarioId}/calibration`)
+        .set('Authorization', `Bearer ${token}`)
+        .send({
+          range: blmRange,
+        })
+        .expect(HttpStatus.CREATED),
+    ThenCalibrationResultsShouldBeAvailableViaAPI: async () => {
+      const response: { body: BlmCalibrationRunResultDto[] } = await request(
+        app.getHttpServer(),
+      )
+        .get(`/api/v1/scenarios/${scenarioId}/calibration`)
+        .set('Authorization', `Bearer ${token}`)
+        .expect(200);
+
+      const results = response.body;
+
+      expect(results.length).toBeGreaterThan(0);
+
```

Comment:
another thing we may want to check here is that the blm values follow a monotonic sequence.

This would actually be a test that checks the BLM value calculator results are correct (given the `[min, max]` bounds, the six generated values are on a monotonic sequence), so maybe actually best to ignore me on this one 😅

Likewise, in practice that `result.blmValue` is within the bounds of `blmRange` would be something to include in tests for the BLM values set calculator, but they can stay here too if they are not testing for this elsewhere.

Reply:
I think that tests in `api/apps/api/src/modules/projects/blm/domain/blm-values-calculator.spec.ts` ([link](https://github.com/Vizzuality/marxan-cloud/blob/9ecda96ef097a80a8ceb812826cbe60d1a048b26/api/apps/api/src/modules/projects/blm/domain/blm-values-calculator.spec.ts)) already ensures that resulting blm values are between the given range and follow a monotonic sequence

Reply:
yes, that's all we need, indeed 👍🏼